image_explorer/public/css/image_explorer.css

```diff
@@ -94,3 +101,28 @@ div.image-explorer-hotspot .image-explorer-hotspot-reveal .image-explorer-hotspo
     padding-left: 10px;
     color: red;
 }
+
+@media only screen and (max-width: 767px){
+    .image-explorer-wrapper div.image-explorer-hotspot{
+        background-size: 100%;
+        width: 28px;
+        height: 28px;
+    }
+    .image-explorer-wrapper div.image-explorer-hotspot:hover{
+        background-position: 0 -30px;
+    }
+    .image-explorer-hotspot-reveal{
+        right: 0;
+        left: 0;
+        top: 10vh;
+        bottom: 10vh;
+        width: auto !important;
+        height: auto !important;
```

Comment:
Are these `!important` required?

Reply:
yes because I had to override inline properties